ui/components/ConfirmationModal.js

```diff
@@ -13,8 +13,8 @@ import { pingKubernetes } from "./ConnectionWizard/helpers/kubernetesHelpers";
 import { getK8sConfigIdsFromK8sConfig } from "../utils/multi-ctx";
 import { bindActionCreators } from "redux";
 import { useEffect, useState } from "react";
-import UndeployIcon from "../public/static/img/UndeployIcon";
 import DoneAllIcon from '@material-ui/icons/DoneAll';
+import RemoveDoneIcon from '@mui/icons-material/RemoveDone';
```

Comment:
it's good to directly use icon from mui but as `UndeployIcon` is also used in other files deleting it can cause an error so please restore it

Reply:
Hi @Yashsharma1911, you are right.  I totally forgot about that. Another alternative could be replacing all the undeploy svg in the file with the mui. The SVG file does not adapt itself based on them. What do you think?

Reply:
yes I think it will be good

Reply:
Is this the intended to use `@mui/*` instead of `@material-ui/*`?

while I understand the packages for `@mui/*` is still within the list of packages, `@mui/*` is not to be used until we've migrated to `@mui/*` from `@material-ui/*`.

Meaning we are still on Material UI v4, not MUI v5.

On a side note, I need to remove these packages from `ui` to avoid further confusion.

Reply:
okay, thanks for the heads up. I will return it back to its normal way.